src/main/java/com/gigacal/controller/CalendarController.java

```diff
@@ -6,11 +6,7 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
I just want to put this comment at the very top -> you HAVE TO check in every single endpoint if the user have the permissions to do certain things. In your case - is the logged in user the owner of specific calendar? If not throw an exception - if yes proceed with what you've already implemented